docs/rst/user_manual/layout.rst

```diff
@@ -219,22 +219,26 @@ For the explanation of this information refer to the section :ref:`log_panel`.
 
 Main Panel
 ==========
-The central window it will display the data charts, so-called *Chartbox*, that the user has configured.
-To create a new *Chartbox* go to *Edit->DisplayNewData* in the application menu.
-This button will create a new *Chartbox* where new series could be added, removed or modified.
+The central window allows multiple tabs for different views.
+Also it is displayed a collapsed menu with the possible problems that have been detected on the DDS entities.
+The tab panel allows multiple tabs generation so different views are contemplated in the same window.
+It will display the data charts, so-called *Chartbox*, that the user has configured.
+It will display a domain graph where the physical, logical and DDS entities from a domain are represented,
+focusing on the connection between endpoints through the topics, and the entities physical inheritance.
 
 .. figure:: /rst/figures/screenshots/main_panel.png
     :align: center
 
-For further information about how to create a chart refer to the section :ref:`chart_panel`.
-
 .. _chartbox_layout:
 
 Chartbox
 --------
-These windows in the central panel store some *series* or *data configurations* to show a specific data type for
+These windows in the main panel store some *series* or *data configurations* to show a specific data type for
 one or several entities in a specified time interval and with different accumulative operations on the data.
 
+To create a new *Chartbox* go to *Chart View* in the Main Panel default tab.
```

Comment:
This instruction creates a new chart view (no more chart views can be created afterwards).